src/front/app/components/globals/Input.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const classNames = require('classnames');
+
+const types = [
+  // Input Based
+  'text',
+  'email',
+  'password',
+  'number',
+  'date',
+  'datetime',
+  'datetime-local',
+  'search',
+  'tel',
+  'color',
+  'file',
+  // Select Based
+  'select',
+  // Text Area Based
+  'textarea',
+];
+
+class Input extends Component {
+  static propTypes = {
+    type: PropTypes.oneOf(types).isRequired,
+    id: PropTypes.string.isRequired,
+    name: PropTypes.string,
+    className: PropTypes.string,
+    children: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.node,
+      PropTypes.instanceOf(Component),
+    ]),
+    rowCount: PropTypes.number,
+  };
+
+  static defaultProps = {
+    type: 'text',
+  };
+
+  render() {
+    const {
+      type,
+      id,
+      name,
+      className,
+      children,
+      rowCount,
```

Comment:
You're extracting it from the object, yet not passing it along to the te tarea, meaning that rowCount would never reach textarea. Remove the prop entirely.